subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "common.h"
+#include "platform_keys/platform_keys.h"
```

Comment:
```suggestion
#ifdef NRF54H_SERIES
#include "platform_keys/platform_keys.h"
#endif
```

This will not work for the L15, you should ifdef that for the 54H devices.